crates/adapters/bybit/src/common/parse.rs

```diff
@@ -114,12 +115,6 @@ where
     s.parse::<u8>().map_err(serde::de::Error::custom)
 }
 
-/// Extracts the raw symbol from a Bybit symbol by removing the product type suffix.
-#[must_use]
-pub fn extract_raw_symbol(symbol: &str) -> &str {
```

Comment:
I don't think we should move this to the common crate.

Symbols won't always normalize in the same way across adapters - these two just happen to be the same. Because the naming is very general, it's then ambiguous whether to use the function from common (can also be hard to discover) or if the raw symbol extraction function was forgotten etc.

For the sake of a single line, I think best to have a standard pattern with a function in the adapter crates `common/parse.rs`.

Reply:
Done, thanks.